srcpkgs/fyi/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'fyi'
+pkgname=fyi
+version=1.0.4
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc"
+makedepends="dbus-devel"
+depends="dbus"
```

Comment:
Runtime depending on dbus feels excessive

Reply:
I don't why it doesn't get picked up as a runtime dep automatically with `dbus-devel` under `makedepends`. Or maybe you mean almost all Void systems have `dbus` installed anyway so it's redundant.

Reply:
It does pick up `dbus-libs` as the common denominator, `dbus` (runtime binaries) isn't strictly necessary, and users are expected to install it manually anyway.

Reply:
Al right, makes sense. I initially put it explicitly since upstream says

> `fyi` has a single run-time dependency: [dbus](https://www.freedesktop.org/wiki/Software/dbus/) (the original D-Bus implementation).

Reply:
> Al right, makes sense. I initially put it explicitly since upstream says
> 
> > `fyi` has a single run-time dependency: [dbus](https://www.freedesktop.org/wiki/Software/dbus/) (the original D-Bus implementation).

FWIW, that's [how Arch is packaging it](https://gitlab.archlinux.org/archlinux/packaging/packages/fyi/-/blob/main/PKGBUILD?ref_type=heads), as well...